crates/viewer/re_ui/src/list_item/list_item.rs

```diff
@@ -287,10 +307,16 @@ impl ListItem {
             force_hovered,
             force_background,
             collapse_openness,
-            height,
+            mut height,
+            y_offset,
             render_offscreen,
         } = self;
 
+        if y_offset != 0.0 {
+            ui.add_space(y_offset);
+            height -= y_offset;
+        }
+
```

Comment:
The result of this is the gap, visible eg here:

<img width="227" alt="image" src="https://github.com/user-attachments/assets/629073db-f419-4696-b762-2599c4113680" />

I wonder if things would not look better if we kept the original height for the highlight.

An alternative would be to not have full-width highlight for those titles, but some other kind of highlight (to suggest they are [sometime] clickable). I think i would prefer that, but then it becomes a design issue, so probably out of scope for this pr.